translations/en.json

```diff
@@ -1,7 +1,3 @@
 {
-  "Active Medications": "Active Medications",
-  "Active Programs": "Active Programs",
-  "Since": "Since",
-  "care programs": "Care Programs",
-  "conditions": "Conditions"
+  "Active Medications": "Active Medications"
```

Comment:
You seemed to have lost a number of translations - was that on purpose? If not, please maintain.

Reply:
placed them back